### PR TITLE
Bring back the scrollbars

### PR DESCRIPTION
**Pull request by tobiasdiez**
Looks now as follows:
image
Fixes https://github.com/JabRef/jabref/issues/5374.

----

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [x] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Does it work with the Dark Theme as well or do we need adjustments there? (regarding color)

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an exception which occurred when closing JabRef. [#5348](https://github.com/JabRef/jabref/issues/5348)
 - We fixed a few problems that prevented JabFox to communicate with JabRef. [#4737](https://github.com/JabRef/jabref/issues/4737) [#4303](https://github.com/JabRef/jabref/issues/4303)
 - We fixed an error where the groups containing an entry loose their highlight color when scrolling. [#5022](https://github.com/JabRef/jabref/issues/5022) 
+- We fixed an error where scrollbar were not shown. [#5374](https://github.com/JabRef/jabref/issues/5374)
```
Comment (Siedlerchr) [inline CHANGELOG.md]:
```suggestion
- We fixed an error where scrollbars were not shown. [#5374](https://github.com/JabRef/jabref/issues/5374)
```

File: src/main/java/org/jabref/gui/maintable/SmartConstrainedResizePolicy.java [hunk 2]
```diff
@@ -42,6 +42,10 @@ private Boolean initColumnSize(TableView<?> table) {
                 for (TableColumnBase col : visibleLeafColumns) {
                     double share = col.getPrefWidth() / totalPrefWidth;
                     double newSize = tableWidth * share;
+
+                    // Just to make sure that we are staying under the total table width (due to rounding errors)
+                    newSize -= 2;
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/maintable/SmartConstrainedResizePolicy.java]: Please extract this magic number to a constant 😼

**Comment by tobiasdiez** [comment]
The dark theme needs further work (and in general needs to be revisted a bit, so beyond this PR)

**Pull request merged**